db/migrate/20151124162901_add_user_data_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserDataToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :residence, :string
+    add_column :users, :street, :string
+    add_column :users, :division, :string
```

Comment:
There are only 13 possible divisions to save so why don't you just use an integer, like division_id, and map that id on DIVISIONS that has been declared in the user model?